Description: return 0 on a duplicate certificate
 Modify openssl rehash so it returns 0 on a duplicate certificate like c_rehash does.
Author: Brian Murray <brian@ubuntu.com>
Origin: vendor
Bug: https://github.com/openssl/openssl/issues/6083
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/openssl/+bug/1764848
Last-Update: 2018-04-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: openssl-1.1.0g/apps/rehash.c
===================================================================
--- openssl-1.1.0g.orig/apps/rehash.c
+++ openssl-1.1.0g/apps/rehash.c
@@ -132,7 +132,7 @@ static int add_entry(enum Type type, uns
             BIO_printf(bio_err,
                        "%s: skipping duplicate %s in %s\n", opt_getprog(),
                        type == TYPE_CERT ? "certificate" : "CRL", filename);
-            return 1;
+            return 0;
         }
         if (strcmp(filename, ep->filename) == 0) {
             found = ep;
